src/main/java/dota/buff/controller/HeroController.java

```diff
@@ -0,0 +1,30 @@
+package dota.buff.controller;
+
+import dota.buff.model.HeroDTO;
+import dota.buff.service.HeroService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("api/v1/hero")
+public class HeroController {
+    private final HeroService heroService;
+
+    @Autowired
+    public HeroController(HeroService heroService) {
+        this.heroService = heroService;
+    }
+
+    @GetMapping("/{id}")
+    public HeroDTO getHeroById(@PathVariable Integer id){
+        return heroService.getHeroById(id);
+    }
+
+    @GetMapping("/all")
+    public List<HeroDTO> getAllHeroes(){
+        return heroService.getAllHeroes();
```

Comment:
Just as a note to remember: we need to cover the case when we catch the exception, but it relates to another ticket.

Reply:
I'd prefer to use ResponseEntity as a returned type, it allows many things e.g. statuses. It would be smth like `return new ResponseEntity<String>(heroService.getAllHeroes(), HttpStatus.OK)` or 
`return ResponseEntity.status(HttpStatus.OK).body(heroService.getAllHeroes())` choose any one which you like(or could google which would be better probobly)